cmd/format/format.go

```diff
@@ -129,27 +129,36 @@ func Run(v *viper.Viper, statz *stats.Stats, cmd *cobra.Command, paths []string)
 	// checks all paths are contained within the tree root and exist
 	// also "normalize" paths so they're relative to cfg.TreeRoot
 	for i, path := range paths {
-		absolutePath, err := filepath.Abs(path)
-		if err != nil {
-			return fmt.Errorf("error computing absolute path of %s: %w", path, err)
+		absolutePath, e := filepath.Abs(path)
+		if e != nil {
+			return fmt.Errorf("error computing absolute path of %s: %w", path, e)
 		}
 
-		relativePath, err := filepath.Rel(cfg.TreeRoot, absolutePath)
-		if err != nil {
-			return fmt.Errorf("error computing relative path from %s to %s: %w", cfg.TreeRoot, absolutePath, err)
+		if walkType != walk.Stdin {
+			if _, e = os.Stat(absolutePath); e != nil {
+				return fmt.Errorf("path %s not found", path)
+			}
+
+			// // symlinks are allowed on `paths` input, we resolve them here, since
+			// // the readers will k
+			// realPath, ee := filepath.EvalSymlinks(absolutePath)
```

Comment:
@brianmcgee : I tried here to test the `filepath.EvalSymlinks` (and then remove the `Stat` because `EvalSymlinks` should do that already) to resolve symlinks but 
the function does error out in tests? I dont get it: It does not find the file?